packages/fluent/scss/daterangepicker/_layout.scss

```diff
@@ -0,0 +1,39 @@
+@use "../core/" as *;
+@use "../input/_variables.scss" as *;
+@use "_variables.scss" as *;
+
+@mixin kendo-daterangepicker--layout() {
+
+    // Daterange picker
+    .k-daterange-picker {
+        width: var( --kendo-daterange-picker-width, #{$kendo-input-width} );
```

Comment:
I think here we should add var which is equal to the kendo-input-width:

var( --kendo-input-width, #{$kendo-input-width} )

In this way we gain two things:
1. the width of the daterangepicker can be changed with the Sass variable $kendo-daterange-picker-width, not only with the CSS var
2. If the --kendo-input-width css var is globally changed, it'll apply here as well